backend/routes/index.js

```diff
@@ -38,4 +46,28 @@ router.get( '/calendar/:calendar', async ( req, res ) => {
    }
 } );
 
+router.put( '/calendar/:calendar', async ( req, res ) => {
+   try {
+      const calendarUri = `${req.params.calendar}@group.calendar.google.com`;
+      const bodyElements = [ 'startDate', 'endDate', 'summary', 'description' ];
+      const body = _.pick( req.body, bodyElements );
+      _.forEach( bodyElements, ( el ) => assertPresent( body[el] ) );
```

Comment:
It looks like you could use yup or joi here to validate request body. Take a look at these packages, maybe you'll find them useful.

Reply:
Fixed in #46 .